### ISSUE TITLE
cloud docs: would be useful to know what region the cloud instance is in

### ISSUE THREAD

**Issue by mjibson** [MaterializeInc/materialize#7159]
When using s3 sources with mz cloud, I need to know what region the mz instance is in to reduce egress costs.

**Comment by awang** [comment]
there isn't a great place to add this info (I believe we are in us-east today).  Perhaps we should add a new FAQ or “operations” page?

**Comment by mjibson** [comment]
Yes. It'd be nice if it also had some basic instance specs like vCPUs and RAM.

**Comment by nmeagan11** [comment]
We could add this to the [cloud account limits page](https://materialize.com/docs/cloud/account-limits/), which already includes the default specs for vCPUs and RAM.
